dotCMS/src/main/java/com/dotmarketing/portlets/workflows/ajax/WfStepAjax.java

```diff
@@ -1,5 +1,9 @@
 package com.dotmarketing.portlets.workflows.ajax;
 
+import com.dotcms.business.WrapInTransaction;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.DeserializationFeature;
+import com.dotcms.repackage.com.fasterxml.jackson.databind.ObjectMapper;
+import com.dotcms.workflow.form.WorkflowActionStepForm;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotcms.workflow.form.WorkflowActionStepForm'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)